test/powershell/Language/Scripting/I18n.Tests.ps1

```diff
@@ -52,6 +52,7 @@ Describe 'Testing of script internationalization' -Tags "CI" {
 
         import-localizedData mydata -uiculture nl-NL -ea SilentlyContinue -ev ev
 
+        $ev | Should Not BeNullOrEmpty
         $ev[0].Exception.GetType() | Should Be System.Management.Automation.PSInvalidOperationException
```

Comment:
You use `$result | Should BeOfType` in some places, and `$result | Should Not BeNullOrEmpty` then `$result.Exception.GetType | Should Be` in some other places. Why don't you unify them with a single pattern?